### DISCUSSION TITLE
Inspector vue2-vuex-inspector not found

### DISCUSSION THREAD

**Discussion by jfrueh** [vuejs/devtools-v6#1970] [General]
I'm not sure the best place to put this question - I don't think it's a bug, but I'm hoping to get some assistance.

Recently performance has degraded to the point where I can't use the tools.  I'm using the Chrome extension (v 6.4.4) on Windows.   I know that I should be creating a performance profile for you to best analyze this so I went through all of the steps to create it (super easy by the way - nice documentation), but when I run my app locally the vue devtools just freeze and never open, so my performance is not impacted at all.  For that reason, I'm unable to provide a performance profile that contains the issue.

I've tried all of the normal tips of disabling performance monitoring, keeping 'update tracking' disabled, etc. but with no luck.  

This is a big project and hard for me to make available as a repro (although I will explore trying to do it if needed).  I do see 2 warnings in my console after I turn 'Debugging Info' on in the global settings.  These errors are:

**Inspector vue2-vuex-inspector not found**
**Timeline layer vue2-vuex-mutations not found**

Does anybody have any thoughts on what these 2 warnings might be from and how to resolve them?

I believe this issue is related to something with vuex (I'm using vuex 3.4.0 & vue 2.6.11).  

Thanks for any help or direction you can give.

**Comment by duyhachix** [comment]
![image](https://user-images.githubusercontent.com/107833418/206089786-75ea63fb-f857-4021-941e-bab27d80b4f3.png)
Try check in TimeLine option at vue devtool instead of Vuex, the first time I was unable to see timeline in Mutations. And I just found it out by accident

**Comment by jfrueh** [comment]
I actually found my issue with this.  I had an object in my vuex store that was large (an array with ~ 1,000 elements).  Each element was just s simple object with around 10 properties).  As soon as I truncated this array to only contain 100 elements everything worked perfectly again.  So it's a performance issue with a large array in the store when dev tools is open.
